app/Libraries/Search/BeatmapsetSearch.php

```diff
@@ -129,6 +130,15 @@ private function addExtraFilter($query)
         }
     }
 
+    private function addBundledFilter($query)
+    {
+        if ($this->params->showBundled) {
+            $query->filter(['ids' => [
+                'values' => config('osu.beatmapset.client_bundle'),
```

Comment:
this setting is the ids of the current client bundle; the proposal was for all bundled maps, i.e. including maps previously bundled but no longer bundled - this should be stored in the database.

This setting is also no longer used nor updated and will be removed